resources/lib/navigation/directory_search.py

```diff
@@ -38,6 +38,7 @@ def route_search_nav(pathitems, perpetual_range_start, dir_update_listing, param
     path = pathitems[2] if len(pathitems) > 2 else 'list'
     LOG.debug('Routing "search" navigation to: {}', path)
     ret = True
+    dir_update_listing = True if params.get('container_update') else dir_update_listing
```

Comment:
this could be useful in future for other things,
then change in this way:
move this code to directory.py, to replace:
https://github.com/CastagnaIT/plugin.video.netflix/blob/master/resources/lib/navigation/directory.py#L49
with:
`

        if 'dir_update_listing' in self.params:
            self.dir_update_listing = self.params['dir_update_listing'] == 'True'
        else:
            self.dir_update_listing = bool(self.perpetual_range_start)

`